### PR TITLE
Add roundSlider w/ git auto-update via single package.json

### PR DESCRIPTION
**Pull request by dakshshah96**
Pull request for issue: #10110
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/soundar24/roundSlider
 * Official website (optional, not the repository): http://roundsliderui.com/
 * NPM package url (optional): 
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 13
   - Count of stars: 113
   - Count of forks: 33
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/roundSlider/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "license": "MIT",
+  "author": "Soundara Pandian <@soundar24>",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/soundar24/roundSlider.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "roundslider*"
```
Comment (hare1039) [inline ajax/libs/roundSlider/package.json]: Although this rule work, we want all the files under `dist` folder.
So can you change to `**/*`?

File: ajax/libs/roundSlider/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "roundSlider",
+  "filename": "roundslider.min.js",
+  "description": "A free jQuery plugin",
```
Comment (hare1039) [inline ajax/libs/roundSlider/package.json]: A little bit strange in here.
But it's same as upstream, I think it is ok.

**Comment by hare1039** [comment]
Also your master branch seems too old, is it convenient for you to rebase the branch?

Maybe this could help you https://github.com/cdnjs/cdnjs/blob/master/documents/sparseCheckout.md

**Comment by dakshshah96** [comment]
@hare1039 I can pretty much understand the instructions for [sparseCheckout](https://github.com/cdnjs/cdnjs/blob/master/documents/sparseCheckout.md). What are the suggested rules for sparse-checkout? Should I just add the `ajax/libs/library-name`, where `library-name` denotes the library I'm working on? Will that be enough?

**Comment by hare1039** [comment]
```
/build
/documents
/test
/spec
/tools
/LICENSE
/hooks
/*.yml
/*.md
/*.json
/*.sh
/*.js
/.jscsrc
/ajax/libs/${LIBRARY_NAME}/*
```
is basically OK.

**Comment by dakshshah96** [comment]
@hare1039 I've added the rules to my sparse-checkout file and it now looks like this:
screenshot 2017-04-01 16 34 24
After adding the rules, I did `git reset --hard` as specified [here](https://github.com/cdnjs/cdnjs/blob/master/documents/sparseCheckout.md). However, I don't see any folder `ajax` there even though I've added it to my sparse-checkout file.
screenshot 2017-04-01 16 34 47

**Comment by hare1039** [comment]
You can pull the branch from dakshshah96/cdnjs and than checkout to the branch. I think the folder will appear.

**Comment by PeterDaveHello** [comment]
@dakshshah96 you need to checkout to branch `add/roundslider`

**Comment by dakshshah96** [comment]
@PeterDaveHello @hare1039 I've managed to successfully pull changes from master and update my forked repository. I've also pulled my branch `add/roundslider`. How do I rebase the branch so this pull request doesn't remain old anymore?

**Comment by PeterDaveHello** [comment]
`git rebase master add/roundslider`

**Comment by hare1039** [comment]
or `git checkout add/roundslider && git rebase master`
than `git push your-fork add/roundslider -f`.
the `-f` may need because it overwrites remote branch.

**Comment by PeterDaveHello** [comment]
wow ... what's going on here ... let me fix it ...

**Comment by PeterDaveHello** [comment]
@dakshshah96 have you cloned the repository to local and can update the commit message? Thanks.

**Comment by dakshshah96** [comment]
Hi @PeterDaveHello, no I haven't yet. Would you please update the commit message? Thanks! 💯

**Comment by PeterDaveHello** [comment]
@dakshshah96 Okay.  @sufuf3 take it.

**Comment by sufuf3** [comment]
@clairetsai818 @hare1039 please help review this PR, thanks.

**Pull request merged**